chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CertifiedProductSearchDetails.java

```diff
@@ -30,6 +32,7 @@
 @XmlType(namespace = "http://chpl.healthit.gov/listings")
 @XmlAccessorType(XmlAccessType.FIELD)
 @JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
```

Comment:
There are some fields that have null values for certified products but are expected by the front end; will this annotation remove those?

Reply:
Yeah, probably. I wasn't aware that this object is used in many places, not just the controller. So, with that in mind, I think I'm going to refactor this to use a new object that doesnt have those fields and have the controller methods in question return that object.